table/multiget_context.h

```diff
@@ -11,6 +11,7 @@
 #include "db/dbformat.h"
 #include "db/lookup_key.h"
 #include "db/merge_context.h"
+#include "rocksdb/db.h"
```

Comment:
Can this one be more specific, like `options.h`? We prefer not to have "table" code depend on "db" code.